editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -835,14 +802,56 @@ export function createWrapInGroupActions(
 
         const foundMetadata = MetadataUtils.findElementByElementPath(metadata, target)
 
-        acc.push(...createPinChangeCommandsForElement(foundMetadata, expectedPathInsideGroup))
+        acc.push(
+          ...createPinChangeCommandsForElement(
+            foundMetadata,
+            expectedPathInsideGroup,
+            globalBoundingBoxOfAllElementsToBeWrapped,
+            newLocalRectangleForGroup,
+          ),
+        )
       })
     })
   })
 
   return applyCommandsAction([...deleteCommands, insertGroupCommand, ...pinChangeCommands])
 }
 
+export function createPinChangeCommandsForElement(
```

Comment:
now that this is exported, I think the name should refer to elements that are now becoming group children... maybe it's too naive but how about `createPinChangeCommandsForElementBecomingGroupChild` ?

Reply:
I guess it'd work! https://github.com/concrete-utopia/utopia/pull/3993/commits/1579c5e1628672cfebc4a9e31b2cb0f457976165